tsptw_data_dt.h

```diff
@@ -367,6 +375,24 @@ class TSPTWDataDT {
           RoutingIndexManager::NodeIndex(vehicle_indices[j.value()]));
     }
 
+    std::vector<int64> MaxTimes(int32 size_matrix) const {
+      int64 max_row;
+      std::vector<int64> max_times;
+      for (int32 i = 0; i < size_matrix; i++) {
+        max_row = 0;
+        for (int32 j = 0; j < size_matrix; j++) {
+          if (data->times_matrices_[problem_matrix_index].Cost(
+                  RoutingIndexManager::NodeIndex(i), RoutingIndexManager::NodeIndex(j)) <
+              CUSTOM_MAX_INT)
+            max_row = std::max(max_row, data->times_matrices_[problem_matrix_index].Cost(
```

Comment:
Like I said, vehicles have the original `ortools_vrp::Matrix& matrix`. If we implement it like this, I need to reimplement it from scratch to fix the conflict. That is the reason I mention specifically using `ortools_vrp::Matrix& matrix` in my previous request.